ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -126,7 +126,7 @@ def backup(host):
 
     # Get the backup location from the command's output
     for line in result.stderr_text.splitlines():
-        prefix = ('ipa.ipaserver.install.ipa_backup.Backup: '
+        prefix = ('ipaserver.install.ipa_backup: '
```

Comment:
This code segment is broken and does not what you think it is doing. You are passing a tuple to ``startswith``. That's a multi-match. That's totally fine. Next up you are using the length of the **tuple** to strip the prefix from the line -- which is always 2 here. You want the length of the matching **string** instead.

Reply:
Oh, it's not a tuple but a long string within parenthesis... Can you fit the entire string on one line?

Reply:
Sure, yes